### DISCUSSION TITLE
Calendar Hearmap Color Reference

### DISCUSSION THREAD

**Discussion by neinja007** [monkeytypegame/monkeytype#5661] [Questions] [answered]
Hello,

Can someone inform me about how many activities need to be run to achieve a certain level of activity on the heatmap below?

![image](https://github.com/user-attachments/assets/b980983f-e766-4d1b-990f-0dd36541a0b6)

**Comment by fehmer** [comment]
levels are dynamic based on your activity

**Reply by neinja007** [reply]
i know that. my question was how many activities were required to reach a certain color.

**Reply by fehmer** [reply, marked as answer]
We take the average of all days except 10% of the days with the lowest and highest count each. 

The first color is for days without any activity. The other 4 levels are under 25%, 50%, 75% and over 75% of the average.
